src/screens/Vault/components/WalletConfiguration.jsx

```diff
@@ -92,15 +92,7 @@ function WalletConfiguration({
     },
     {
       id: 2,
-      label: `${isIos ? 'Airdrop / ' : ''}File Export`,
-      icon: <AirDropIcon />,
-      onPress: () => {
-        shareWithAirdrop();
-      },
-    },
-    {
-      id: 3,
-      label: 'Show QR',
+      label: 'Scan QR',
```

Comment:
Why was this changed? Show QR was correct. Should use Show QR when showing a QR and Scan QR when showing the scanner